src/stress/adapter/device_allocation.ts

```diff
@@ -0,0 +1,28 @@
+export const description = `
+Stress tests for GPUAdapter.requestDevice.
+`;
+
+import { makeTestGroup } from '../../common/framework/test_group.js';
+import { GPUTest } from '../../webgpu/gpu_test.js';
+
+export const g = makeTestGroup(GPUTest);
```

Comment:
(For better or worse,) GPUTests always provide their own GPUDevice object. This is fine, but since it won't specifically be used these could also use the base `Fixture` fixture or something.

Reply:
Done